src/components/policy/policy_external/src/sql_pt_representation.cc

```diff
@@ -1412,6 +1415,12 @@ bool SQLPTRepresentation::SaveModuleConfig(
   config.certificate.is_initialized() ? query.Bind(9, *(config.certificate))
                                       : query.Bind(9);
 
+  if (config.lock_screen_dismissal_enabled.is_initialized()) {
```

Comment:
Prefer the conditional form, as is used in the rest of the function:
```
config.lock_screen_dismissal_enabled.is_initialized() ? query.Bind(10, *(config.lock_screen_dismissal_enabled))
                                      : query.Bind(10);
```

Reply:
@jacobkeeler Thanks. Fixed in 99cf2e13b8511db9c20bc0feebcca0d1610e075f